ZipBuilder/Sources/UpdateFirebasePod/FirebasePod.swift

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+import ManifestReader
+
+/// Updates the Firebase Pod with a release's version set.
+struct FirebasePod {
+
+  /// Relevant paths in the filesystem to build the release directory.
+  struct FilesystemPaths {
+    // MARK: - Required Paths
+
+    /// A file URL to a textproto with the contents of a `FirebasePod_Release` object. Used to verify
+    /// expected version numbers.
+    var currentReleasePath: URL?
```

Comment:
Hmm, I didn't change this from ZipBuilder.swift. How should variables defined that are later set by options?

Reply:
Ahh okay - so in the ZipBuilder it's optional since you can build without it being a release candidate (i.e. build from public pods) but for this tool it's a required one.

You can define this as `let currentReleasePath: URL` and then as long as all initializers assign it before returning, that's valid. In our case, we'd throw a `fatalError` if the path isn't provided so it works with this pattern.